esmvaltool/cmorizers/data/formatters/datasets/cds_satellite_soil_moisture.ncl

```diff
@@ -43,7 +43,7 @@ loadscript(getenv("esmvaltool_root") + \
 begin
```

Comment:
That script is not working because the path to `interface.ncl` has not been changed

Reply:
Fixed